### PR TITLE
Adds GA event for email signup

### PR DESCRIPTION
**Pull request by shawyu-okta**
## Description:
- **What's changed?** 
  * Adds GA event for email signup success
  * Event will be published to GTM on Tuesday 1/26 10am PST but we can merge and deploy to production any time before then

### Resolves:

* [OKTA-362748](https://oktainc.atlassian.net/browse/OKTA-362748)

### CODE REVIEW

**Comment by jakubvul-okta** [comment]
Ack

File: packages/@okta/vuepress-theme-prose/global-components/SignUp.vue [hunk 1]
```diff
@@ -334,11 +334,16 @@ export default {
 
         this.isPending = true;
 
+        /*
         this.apiService
           .post(registrationPath, { body })
+          */
+        Promise.resolve()
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/global-components/SignUp.vue:341]: temporarily skip the API for testing

File: packages/@okta/vuepress-site/.vuepress/config.js [hunk 2]
```diff
@@ -26,8 +26,9 @@ module.exports = {
      * Header scripts for typekit, GA, GTM (WIP)
      */
     ['script', {}, `
+      window.dataLayer = window.dataLayer || [];
 
-      var isProduction = window.location.hostname === 'developer.okta.com';
+      var isProduction = true;//window.location.hostname === 'developer.okta.com';
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-site/.vuepress/config.js:31]: Temporarily force GTM to load

**Review by kamaldeepcheema-okta — APPROVED** [review]

**Comment by shawyu-okta** [comment]
Event details have been signed off by Susan Sheng on analytics.

**Review by francoislehoux-okta — APPROVED** [review]

**Pull request merged**